std/range/package.d

```diff
@@ -7436,11 +7457,12 @@ auto only(Values...)(auto ref Values values)
 {
     import std.algorithm.comparison : equal;
     import std.algorithm.iteration : filter, joiner, map;
-    import std.algorithm.searching : findSplitBefore;
+    import std.algorithm.searching : find;
     import std.uni : isUpper;
 
-    assert(equal(only('♡'), "♡"));
-    assert([1, 2, 3, 4].findSplitBefore(only(3))[0] == [1, 2]);
```

Comment:
BTW This example is moving to: https://github.com/dlang/phobos/pull/4842

The `find` one is more general as it doesn't return a custom tuple.